test/TestProjects/FirstTest-Typespec/Generated/Docs/CadlFirstTestClient.xml

```diff
@@ -1421,6 +1479,14 @@ var data = new {
         "1"
     },
     stringFixedEnum = "1",
+    requiredUnknown = new {},
```

Comment:
Can this line work for BinaryData?  VS reports error for this.

Reply:
No. This is an issue in sample generator of DPG, we have tons of similar issues like this. We will be tracking this there. @chunyu3 is summarizing the issues we saw.

If you expand the existing code above, you will see more `new {}`

Reply:
Do we have an issue to track this?  Please link it here and resolve this comment.   @ArcturusZhang

Reply:
The example issues are tracking here: https://github.com/Azure/autorest.csharp/issues/3274